inference-engine/tests/functional/plugin/gpu/batched_blob_preprocess_tests/batched_blob_preprocessing_test.cpp

```diff
@@ -0,0 +1,143 @@
+    }
+
+    // ------------------------------------------------------
+    // execute BatchedBlob inference
+    if (num_batch == 1) {
+        inf_req_mbatch.SetBlob(net_mbatch.getInputsInfo().begin()->first, full_blob[0]);
+    } else {
+        auto batched_blob = make_shared_blob<BatchedBlob>(full_blob);
+        inf_req_mbatch.SetBlob(net_mbatch.getInputsInfo().begin()->first, batched_blob);
+    }
+
+    inf_req_mbatch.Infer();
+
+    auto outputBlob_shared = inf_req_mbatch.GetBlob(net_mbatch.getOutputsInfo().begin()->first);
+
+    // ------------------------------------------------------
+    // Setup another network to inference using batch_size = 1
+    auto fn_ptr_local_sbatch = ngraph::builder::subgraph::makeConvPoolRelu({1, 3, convolution_height, convolution_width});
+
+    CNNNetwork net_sbatch(fn_ptr_local_sbatch);
+
+    net_sbatch.getInputsInfo().begin()->second->setLayout(Layout::NCHW);
+    net_sbatch.getInputsInfo().begin()->second->setPrecision(Precision::U8);
+    net_sbatch.getInputsInfo().begin()->second->getPreProcess().setColorFormat(ColorFormat::NV12);
+    net_sbatch.getInputsInfo().begin()->second->getPreProcess().setResizeAlgorithm(ResizeAlgorithm::RESIZE_BILINEAR);
+    net_sbatch.getOutputsInfo().begin()->second->setPrecision(Precision::FP32);
+
+    auto exec_net_sbatch = ie.LoadNetwork(net_sbatch, CommonTestUtils::DEVICE_GPU);
+
+    auto inf_req_sbatch = exec_net_sbatch.CreateInferRequest();
+
+    // ------------------------------------------------------
+    // Run regular input for each image and compare against batched blob
+    for (int i = 0; i < num_batch; i++) {
+        auto y_blob = make_shared_blob<uint8_t>(y_plane_desc, fake_image_data_y[i]->buffer().as<uint8_t *>());
+        auto uv_blob = make_shared_blob<uint8_t>(uv_plane_desc, fake_image_data_uv[i]->buffer().as<uint8_t *>());
+        auto blob = make_shared_blob<NV12Blob>(y_blob, uv_blob);
+        inf_req_sbatch.SetBlob(net_sbatch.getInputsInfo().begin()->first, blob);
+        inf_req_sbatch.Infer();
+        auto output_blob_local = inf_req_sbatch.GetBlob(net_sbatch.getOutputsInfo().begin()->first);
+
+        // ------------------------------------------------------
+        // This network generates [1, size] tensor whether batch=1 or 2. So need to split
+        auto split_shared_blob = make_shared_blob<float_t>(output_blob_local->getTensorDesc(),
+                                    outputBlob_shared->buffer().as<float_t *>() + output_blob_local->size() * i);
+        ASSERT_EQ(output_blob_local->size(), split_shared_blob->size());
+        float thr = 0.1;
+
+        FuncTestUtils::compareBlobs(output_blob_local, split_shared_blob, thr, "", false);
+    }
+}
+
+const std::vector<size_t> num_batches{1, 2, 4, 5, 6};
+
+INSTANTIATE_TEST_CASE_P(smoke_PreprocessBatchedBlob, IEBatchedBlob_Test, ::testing::ValuesIn(num_batches), IEBatchedBlob_Test::getTestCaseName);
```

Comment:
@ilya-lavrenov I remember your PR, where you switch to official gtest repository and where `INSTANTIATE_TEST_CASE_P` was replaced with something else. Was it merged?

Reply:
Yes https://github.com/openvinotoolkit/openvino/pull/6286

Reply:
changed